src/main/java/com/mraof/minestuck/util/MSBlockSetType.java

```diff
@@ -0,0 +1,53 @@
+	public static final BlockSetType CINDERED = register(new BlockSetType(Minestuck.MOD_ID + ":cindered"));
+	
+	//ASPECT WOOD
+	public static final BlockSetType BLOOD = register(new BlockSetType(Minestuck.MOD_ID + ":blood"));
+	public static final BlockSetType BREATH = register(new BlockSetType(Minestuck.MOD_ID + ":breath"));
+	public static final BlockSetType DOOM = register(new BlockSetType(Minestuck.MOD_ID + ":doom"));
+	public static final BlockSetType HEART = register(new BlockSetType(Minestuck.MOD_ID + ":heart"));
+	public static final BlockSetType HOPE = register(new BlockSetType(Minestuck.MOD_ID + ":hope"));
+	public static final BlockSetType LIFE = register(new BlockSetType(Minestuck.MOD_ID + ":life"));
+	public static final BlockSetType LIGHT = register(new BlockSetType(Minestuck.MOD_ID + ":light"));
+	public static final BlockSetType MIND = register(new BlockSetType(Minestuck.MOD_ID + ":mind"));
+	public static final BlockSetType RAGE = register(new BlockSetType(Minestuck.MOD_ID + ":rage"));
+	public static final BlockSetType SPACE = register(new BlockSetType(Minestuck.MOD_ID + ":space"));
+	public static final BlockSetType TIME = register(new BlockSetType(Minestuck.MOD_ID + ":time"));
+	public static final BlockSetType VOID = register(new BlockSetType(Minestuck.MOD_ID + ":void"));
+	
+	//STONE
+	public static final BlockSetType CRUXITE = register(new BlockSetType(Minestuck.MOD_ID + ":cruxite"));
+	public static final BlockSetType URANIUM = register(new BlockSetType(Minestuck.MOD_ID + ":uranium"));
+	public static final BlockSetType COARSE_STONE = register(new BlockSetType(Minestuck.MOD_ID + ":coarse_stone"));
+	public static final BlockSetType SHADE_STONE = register(new BlockSetType(Minestuck.MOD_ID + ":shade_stone"));
+	public static final BlockSetType CAST_IRON = register(new BlockSetType(Minestuck.MOD_ID + ":cast_iron"));
+	public static final BlockSetType MYCELIUM_STONE = register(new BlockSetType(Minestuck.MOD_ID + ":mycelium_stone"));
+	public static final BlockSetType BLACK_STONE = register(new BlockSetType(Minestuck.MOD_ID + ":black_stone"));
+	public static final BlockSetType CHALK = register(new BlockSetType(Minestuck.MOD_ID + ":chalk"));
+	public static final BlockSetType PINK_STONE = register(new BlockSetType(Minestuck.MOD_ID + ":pink_stone"));
+	public static final BlockSetType BROWN_STONE = register(new BlockSetType(Minestuck.MOD_ID + ":brown_stone"));
+	public static final BlockSetType BLACK_CHESS = register(new BlockSetType(Minestuck.MOD_ID + ":black_chess"));
+	public static final BlockSetType DARK_GRAY_CHESS = register(new BlockSetType(Minestuck.MOD_ID + ":dark_gray_chess"));
+	public static final BlockSetType LIGHT_GRAY_CHESS = register(new BlockSetType(Minestuck.MOD_ID + ":light_gray_chess"));
+	public static final BlockSetType WHITE_CHESS = register(new BlockSetType(Minestuck.MOD_ID + ":white_chess"));
```

Comment:
I don't think these should be using wood sound events. Try using the other `BlockSetType` constructor.

Reply:
Updated. I copied most of them from stone [the item] and copied cast iron from iron. (Which apparently uses the stone button sound events.)